builtins.go

```diff
@@ -1980,6 +2002,209 @@ func builtinManifestJSONEx(i *interpreter, arguments []value) (value, error) {
+		return nil, err
+	}
+	vQuoteKeys, err := i.getBoolean(arguments[2])
+	if err != nil {
+		return nil, err
+	}
+
+	var buf bytes.Buffer
+
+	var aux func(ov value, buf *bytes.Buffer, cindent string) error
+	aux = func(ov value, buf *bytes.Buffer, cindent string) error {
+		switch v := ov.(type) {
+		case *valueNull:
+			buf.WriteString("null")
+		case *valueBoolean:
+			if v.value {
+				buf.WriteString("true")
+			} else {
+				buf.WriteString("false")
+			}
+		case valueString:
+			s := v.getGoString()
+			if s == "" {
+				buf.WriteString(`""`)
+			} else if strings.HasSuffix(s, "\n") {
+				s := strings.TrimSuffix(s, "\n")
+				buf.WriteString("|")
+				for _, line := range strings.Split(s, "\n") {
+					buf.WriteByte('\n')
+					buf.WriteString(cindent)
+					buf.WriteString(yamlIndent)
+					buf.WriteString(line)
+				}
+			} else {
+				buf.WriteString(unparseString(s))
+			}
+		case *valueNumber:
+			buf.WriteString(strconv.FormatFloat(v.value, 'f', -1, 64))
+		case *valueArray:
+			if v.length() == 0 {
+				buf.WriteString("[]")
+				return nil
+			}
+			for ix, elem := range v.elements {
+				if ix != 0 {
+					buf.WriteByte('\n')
+					buf.WriteString(cindent)
+				}
+				thunkValue, err := elem.getValue(i)
+				if err != nil {
+					return err
+				}
+				buf.WriteByte('-')
+
+				if v, isArr := thunkValue.(*valueArray); isArr && v.length() > 0 {
+					buf.WriteByte('\n')
+					buf.WriteString(cindent)
+					buf.WriteString(yamlIndent)
+				} else {
+					buf.WriteByte(' ')
+				}
+
+				prevIndent := cindent
+				switch thunkValue.(type) {
+				case *valueArray, *valueObject:
+					cindent = cindent + yamlIndent
+				}
+
+				if err := aux(thunkValue, buf, cindent); err != nil {
+					return err
+				}
+				cindent = prevIndent
+			}
+		case *valueObject:
+			fields := objectFields(v, withoutHidden)
+			if len(fields) == 0 {
+				buf.WriteString("{}")
+				return nil
+			}
+			sort.Strings(fields)
+			for ix, fieldName := range fields {
+				fieldValue, err := v.index(i, fieldName)
+				if err != nil {
+					return err
+				}
+
+				if ix != 0 {
+					buf.WriteByte('\n')
+					buf.WriteString(cindent)
+				}
+
+				keyStr := fieldName
+				if vQuoteKeys.value || !yamlBareSafe(fieldName) {
+					keyStr = escapeStringJson(fieldName)
+				}
+				buf.WriteString(keyStr)
+				buf.WriteByte(':')
+
+				prevIndent := cindent
+				if v, isArr := fieldValue.(*valueArray); isArr && v.length() > 0 {
+					buf.WriteByte('\n')
+					buf.WriteString(cindent)
+					if vindentArrInObj.value {
+						buf.WriteString(yamlIndent)
+						cindent = cindent + yamlIndent
+					}
+				} else if v, isObj := fieldValue.(*valueObject); isObj {
+					if len(objectFields(v, withoutHidden)) > 0 {
+						buf.WriteByte('\n')
+						buf.WriteString(cindent)
+						buf.WriteString(yamlIndent)
+						cindent = cindent + yamlIndent
+					} else {
+						buf.WriteByte(' ')
+					}
+				} else {
+					buf.WriteByte(' ')
+				}
+				aux(fieldValue, buf, cindent)
```

Comment:
Note that this is unchecked.

Fix: https://github.com/google/go-jsonnet/pull/800

Reply:
This PR (#742) introduces a bug into manifestYamlDoc, which is now part of the 0.21.0 release. (We were even getting stack traces, but that's besides the point here.)

The fix is trivial (#800), but I'm having difficulties getting it merged -- nobody responded...

@jgraeger, can I kindly ask you, as the original author, to help me ping the right people and get the fix merged? Thanks :)